include/sdf/Capsule.hh

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#ifndef SDF_CAPSULE_HH_
+#define SDF_CAPSULE_HH_
+
+#include <ignition/math/Capsule.hh>
+#include <sdf/Error.hh>
+#include <sdf/Element.hh>
+#include <sdf/sdf_config.h>
+
+namespace sdf
+{
+  // Inline bracket to help doxygen filtering.
+  inline namespace SDF_VERSION_NAMESPACE {
+  //
+
+  // Forward declare private data class.
+  class CapsulePrivate;
+
+  /// \brief Capsule represents a capsule shape, and is usually accessed
+  /// through a Geometry.
+  class SDFORMAT_VISIBLE Capsule
+  {
+    /// \brief Constructor
+    public: Capsule();
+
+    /// \brief Copy constructor
+    /// \param[in] _capsule Capsule to copy.
+    public: Capsule(const Capsule &_capsule);
+
+    /// \brief Move constructor
+    /// \param[in] _capsule Capsule to move.
+    public: Capsule(Capsule &&_capsule) noexcept;
+
+    /// \brief Destructor
+    public: virtual ~Capsule();
```

Comment:
is there any point to having a virtual destructor? I copied it from the other classes, but would it be better without the `virtual`?

Reply:
I was going to leave a comment about that but didn't because the other classes use virtual destructors. IMO, these classes are not meant to be used polymorphically (they don't have any virtual functions), so there is no need for a virtual destructor.

Reply:
made non-virtual in 1c09156